components/server/src/server.rs

```diff
@@ -195,13 +195,17 @@ struct TiKVEngines<ER: RaftEngine> {
 
 struct Servers<ER: RaftEngine> {
     lock_mgr: LockManager,
-    server: Server<RaftRouter<RocksEngine, ER>, resolve::PdStoreAddrResolver>,
+    server: ServerType<ER>,
     node: Node<RpcClient, ER>,
     importer: Arc<SSTImporter>,
     cdc_scheduler: tikv_util::worker::Scheduler<cdc::Task>,
     cdc_memory_quota: MemoryQuota,
 }
 
+type ServerType<ER> =
```

Comment:
Use a different name than `xxType`? I think we usually use that for enum.

Reply:
I pushed a change to rename them to LocalServer and LocalRaftKv. Does that work? Hard to come up with a good name for these since they are just here to reduce type complexity.

Reply:
Sounds good.